juicy-select.html

```diff
@@ -42,17 +42,25 @@
                 this.$select.addEventListener("change", this.selectChange.bind(this));
                 this.set("multiple", this.$select.hasAttribute("multiple"));
 
+                this.isAttached = true;
```

Comment:
As far as I know, the `isAttached` is now used by Polymer itself, so element can't use it for it's own purpose.

Reply:
How about we use `this._isAttached` instead so we don't clash with Polymer?